project.clj

```diff
@@ -160,8 +166,8 @@
   ;; Please see:
   ;; https://github.com/bhauman/lein-figwheel/wiki/Using-the-Figwheel-REPL-within-NRepl
 
-  :uberjar-inclusions [".ebextensions"]
-  :jar-inclusions [".ebextensions"]
+  :uberjar-inclusions [#"^\.ebextensions"]
+  :jar-inclusions [#"^\.ebextensions"]
```

Comment:
This is a good find! I did not see it when I was looking for improper regexs for the offending `ClassCastException`.
Out of curiosity, how did you identify this as the issue?

Reply:
found it here: https://github.com/technomancy/leiningen/blob/master/sample.project.clj#L418

Reply:
Just by doing a close compare? I was trying to find it via stacktraces, then just looking for all clojure functions that have regex args. Didn't notice this, myself.

Glad you found it :)